app-arch/unshield/unshield-1.6.2.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="InstallShield CAB file extractor"
+HOMEPAGE="https://github.com/twogood/unshield"
+SRC_URI="https://github.com/twogood/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~hppa ~ppc ~ppc64 ~riscv ~x86"
+
+# OpenSSL is detected at build time, and used to determine
+# whether or not a hand-rolled md5 implementation is used. The build
+# system prefers OpenSSL's implementation if it's available, and OpenSSL
+# is common enough, so we prefer it too. Since the dependency is
+# automagic (there's no way to hide it), we require OpenSSL
+# unconditionally.
+
+RDEPEND="
+	dev-libs/openssl:0=
+"
+DEPEND="${RDEPEND}
+	sys-libs/zlib
```

Comment:
Why isn't zlib in RDEPEND too?

Reply:
According do scanelf it is not linked against the final binary.

Reply:
Never heard of iwdevtools so far. Will integrate them into my workflow for sure.